src/styles/utils/_resets.scss

```diff
@@ -0,0 +1,3 @@
+body {
+  margin: 0;
+}
```

Comment:
im not sure, but its ok to put reset file in the folder "utils"? maybe would be beter outside like main?

Reply:
good point. fixed